resources/lang/en/Debug.properties

```diff
@@ -1,2 +1,4 @@
 filterResponse=Filter response
 FeedbackService_itemDoesNotExistError=The requested item does not exist
+determineVisibilityResponse = Determine visibility response
```

Comment:
Where is this used?

Reply:
Was used for log debugging purposes. I've removed it now.